docs/additional-functionality/rapids-udfs.md

```diff
@@ -174,31 +173,51 @@ To enable _GPU Scheduling for Pandas UDF_, you need to configure your spark job
     --py-files ${SPARK_RAPIDS_PLUGIN_JAR}
     ```
 
-3. Enable GPU Scheduling for Pandas UDF.
+3. Enable GPU Assignment(Scheduling) for Pandas UDF.
 
     ```shell
     ...
     --conf spark.rapids.python.gpu.enabled=true \
-    --conf spark.rapids.python.memory.gpu.pooling.enabled=false \
-    --conf spark.rapids.sql.exec.ArrowEvalPythonExec=true \
-    --conf spark.rapids.sql.exec.MapInPandasExec=true \
-    --conf spark.rapids.sql.exec.FlatMapGroupsInPandasExec=true \
-    --conf spark.rapids.sql.exec.AggregateInPandasExec=true \
-    --conf spark.rapids.sql.exec.FlatMapCoGroupsInPandasExec=true \
-    --conf spark.rapids.sql.exec.WindowInPandasExec=true
     ```
 
-Please note the data transfer acceleration only supports scalar UDF and Scalar iterator UDF currently. 
-You could choose the exec you need to enable.
+Please note the data transfer acceleration only supports `scalar UDF` and `Scalar iterator UDF` currently. 
+You could choose the exec see the section below.
 
 ### Other Configuration
 
 Following configuration settings are also for _GPU Scheduling for Pandas UDF_
-```
-spark.rapids.python.concurrentPythonWorkers
-spark.rapids.python.memory.gpu.allocFraction
-spark.rapids.python.memory.gpu.maxAllocFraction
-```
+
+1. Spark Plan support
+
+    ```
+    --conf spark.rapids.sql.exec.ArrowEvalPythonExec=true \
+    --conf spark.rapids.sql.exec.MapInPandasExec=false \
+    --conf spark.rapids.sql.exec.FlatMapGroupsInPandasExec=false \
+    --conf spark.rapids.sql.exec.AggregateInPandasExec=false \
+    --conf spark.rapids.sql.exec.FlatMapCoGroupsInPandasExec=false \
+    --conf spark.rapids.sql.exec.WindowInPandasExec=true
```

Comment:
Most of these settings match the defaults, so do we need to burden the user with these details?  There's no explanation in this documentation why these settings are necessary.

Reply:
You got reason. I made a table to show the their status at data-transfer part.